generator/writeFile.js

```diff
@@ -59,27 +59,22 @@ function test () {
  */
 
 function splitObject (path,file) {
-  var src = path; // Location of object file
-  var obj = file // Object file name
-  var srclen= src.length;
-  if(src.charAt(srclen-1)!=='/') src = src + '/'; //path correction
-  var location = src + obj // location of object file
-  let ddata = fs.readFileSync(location)
+  if(path.charAt(path.length-1)!=='/') path = path + '/'; //path correction
+  let ddata = fs.readFileSync(path + file)
   let grocery = JSON.parse(ddata)
   var len = grocery.length // Object Length
   var foldername = file.slice(0,-5) ;
-  foldername = foldername+ "_elements";
+  foldername = foldername + "_elements";
   var fileName;
-  if (!fs.existsSync(src + foldername)) fs.mkdirSync(src + foldername) // Check if Folder exists or not.
+  if (!fs.existsSync(path + foldername)) fs.mkdirSync(path + foldername) // Check if Folder exists or not.
```

Comment:
make it in few lines - it's not the best way for reading code

Reply:
done